src/main/resources/templates/pages/add.ftl

```diff
@@ -0,0 +1,18 @@
+<h1>Add new blog</h1>
+<fieldset>
+    <form name="blog" method="post">
```

Comment:
As I can see you use the same form and same errors twice in the project already, in my opinion, it would be better to create a macro called smthg like "editForm" or "blogForm" and put the form and errors there.
Basically, the "edit" can mean both "edit" and "create".